gen/dcos-config.yaml

```diff
@@ -581,6 +1365,12 @@ package:
       MESOS_NETWORK_CNI_PLUGINS_DIR=/opt/mesosphere/active/cni/:/opt/mesosphere/active/dcos-cni/:/opt/mesosphere/active/mesos/libexec/mesos
       MESOS_RECONFIGURATION_POLICY=additive
       MESOS_RECOVERY_TIMEOUT={{ mesos_recovery_timeout }}
+      MESOS_SECCOMP_CONFIG_DIR=/opt/mesosphere/etc/dcos/mesos/seccomp
```

Comment:
I see you put the default seccomp profile `default.json` under `/etc/dcos/mesos/seccomp/`, should it be put under `/opt/mesosphere/etc/dcos/mesos/seccomp` instead? Or actually they are the same directory?

Reply:
Yeah, they are the same.